<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsTab</class>
 <widget class="QWidget" name="OptionsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">OptionsTab</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Language:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="languageComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>News Tab</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="newsMinecraftButton">
        <property name="text">
         <string>Minecraft News</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="newsRssFeedButton">
        <property name="text">
         <string>RSS Feed</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="newsFeedUrlLabel">
          <property name="text">
           <string>URL:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="newsFeedUrlEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="newsWebsiteButton">
        <property name="text">
         <string>Website</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="newsUrlLabel">
          <property name="text">
           <string>URL:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="newsUrlEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="newsFeedDefaultBrowserCheckBox">
        <property name="text">
         <string>Open links in default browser</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="allowMultiLaunchCheckBox">
     <property name="text">
      <string>Allow starting multiple instances (not recommended)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="extractNativesCheckBox">
     <property name="text">
      <string>Unpack natives separately for each instance</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="windowIconProgressCheckBox">
     <property name="text">
      <string>Display installation progress in the window icon</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="flatpakSubsandboxCheckBox">
     <property name="text">
      <string>Launch Minecraft in a Subsandbox</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkUpdatesStartupCheckBox">
     <property name="text">
      <string>Check for Updates on Startup</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Minecraft directory:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="minecraftDirPathButton">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="minecraftDirChangeButton">
       <property name="text">
        <string>Change</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="minecraftDirResetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="displayServerLabel">
       <property name="text">
        <string>Displayserver for Minecraft:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="displayServerBox"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
